doc/index.rst

```diff
@@ -3,15 +3,55 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
+#################################
+ftrack Connect Pipeline Framework
+#################################
+
+
 Welcome to ftrack-connect-pipeline's documentation!
-===================================================
+
+.. important::
+
+    This is the new DCC framework for :term:`Connect` 2 written in :term:`Python`.
+    If you are migrating from the old Connect framework then please read the dedicated
+    :ref:`migration guide <release/migrating_from_legacy_connect_framework>`.
+
+
+This document is intended to bring a deeper understanding of how the new Connect 2
+:term:`framework` works under the hood, enabling developers to customize it to their
+particular needs.
+
+For a general introduction on how to install and use ftrack Connect 2 out of the box,
+please head over here: https://www.ftrack.com/en/portfolio/connect
+
+The documentation is divided into two main parts, the first part is where we describe
+the new Connect 2 framework and the second part is where we customise it to fit our needs.
```

Comment:
Are we shure we want to name it the new Connect2 framework as it was the same thing? or just new framework as it's not strictly related with connect?